postgres-appliance/Dockerfile

```diff
@@ -74,7 +74,7 @@ RUN export DEBIAN_FRONTEND=noninteractive \
         # install etcdctl
         && ETCDVERSION=3.3.27 \
         && curl -L https://github.com/coreos/etcd/releases/download/v${ETCDVERSION}/etcd-v${ETCDVERSION}-linux-$(dpkg --print-architecture).tar.gz \
-                | tar xz -C /bin --strip=1 --wildcards --no-anchored etcdctl etcd \
+                | tar xz -C /bin --strip=1 --wildcards --no-anchored --no-same-owner etcdctl etcd \
         && chown root:root /bin/etcd*; \
```

Comment:
I think this one could be removed.

Reply:
Do you mean, whole etcd could be removed?

Reply:
I mean we don't need `chown`.

Reply:
"chown" has been removed now.
Squashed and amended commits to:
[a4fd10e](https://github.com/zalando/spilo/pull/711/commits/a4fd10ed0b1372d64ba1df411d5acb0e57e9263a)